.circleci/continue_config.yml

```diff
@@ -20,6 +20,11 @@ workflows:
   version: 2
   dist-compile:
     jobs:
+      - native-specific-macos-conda:
+          executor: macos-arm
+      - native-specific-macos-conda:
+          executor: macos-intel
+      - linux-build-and-unit-test-conda
```

Comment:
Note: I haven't tried using `linux-build-and-unit-test-conda` as part of `linux-presto-e2e-tests` and `linux-spark-e2e-tests` jobs below. I've only tested via `ctest`, when run from a conda job itself (see "Run Unit Tests").